RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -128,19 +133,23 @@ public void onError(ErrorContainer errorContainer) {
     }
 
     public static void purchaseProduct(String productIdentifier, String type) {
-        purchaseProduct(productIdentifier, type, null,  0);
+        purchaseProduct(productIdentifier, type, null,  0, false, null);
     }
 
     public static void purchasePackage(String packageIdentifier,
                                        String offeringIdentifier,
                                        @Nullable final String oldSKU,
-                                       final int prorationMode) {
+                                       final int prorationMode,
+                                       final boolean isPersonalized,
+                                       @Nullable final String presentedOfferingIdentifier
```

Comment:
This parameter seems duplicated and unused, there is already an `offeringIdentifier` above, which should be the same I think?

Reply:
Indeed this was duplicated! Since it is used internally only I can remove it without changing the public api